src/SimpleCache.php

```diff
@@ -271,4 +276,32 @@ protected function setValues($values, $ttl): bool
         }
         return $result;
     }
+
+    /**
+     * @param int $defaultTtl
+     */
+    public function setDefaultTtl(int $defaultTtl): void
+    {
+        // @todo validate (possibly everything below 0 is forbidden)
+        $this->defaultTtl = $defaultTtl;
+    }
+
+    /**
+     * @param string $keyPrefix
+     */
+    public function setKeyPrefix(string $keyPrefix): void
+    {
+        if (!empty($keyPrefix) && !ctype_alnum($keyPrefix)) {
```

Comment:
That would forbid names like `cache.name`, `cache_name`, `cache-name`. Is there a technical reason to forbid using any string as a key?

Reply:
Let's remove.